github/packages/github/github_close_issue.js

```diff
@@ -57,10 +57,24 @@ async function _command(params, commandText, secrets = {}) {
       text: body
     });
   } catch (error) {
-    result.push({
-      color: 'danger',
-      text: `Error: ${error.response.status} ${error.response.data.message}`
-    });
+    if (error.response && error.response.status === 403) {

```

Comment:
A suggestion, so that we can factor out common code to a single util file, is to introduce a message generation function:

```
suggestion, a less repetitive way:
```js
if (error.response && error.response.status) {
    result.push({
        text: messageForStatus(error.response.status),
        color: 'danger',
    });
} else ...
```
or even
```
catch (e) {
    result.push({
        text: messageForError(error),
        color: 'danger',
    });
}
```

and share the implementation of `messageForError`.
To be clear, not necessary for this PR but there is a lot of repetition in this command set that should be refactored into common utils.